cg/meta/upload/microsalt/microsalt_upload_api.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+import click
+
+from cg.cli.upload.clinical_delivery import upload_clinical_delivery
+from cg.meta.upload.upload_api import UploadAPI
+from cg.meta.workflow.microsalt.microsalt import MicrosaltAnalysisAPI
+from cg.models.cg_config import CGConfig
+from cg.store.models import Analysis, Case
+
+LOG = logging.getLogger(__name__)
+
+
+class MicrosaltUploadAPI(UploadAPI):
+    def __init__(self, config: CGConfig):
+        self.analysis_api = MicrosaltAnalysisAPI(config)
+        super().__init__(config=config, analysis_api=self.analysis_api)
+
+    def upload(self, ctx: click.Context, case: Case, restart: bool) -> None:
```

Comment:
Feels like this should probably be the default implementation within the parent class? Or is there something MicroSALT specific going on here? Either way, it is out of scope to change the parent class in this PR.